packages/@aws-cdk/core/lib/app.ts

```diff
@@ -118,6 +119,12 @@ export interface AppProps {
    * @default - A `DefaultStackSynthesizer` with default settings
    */
   readonly defaultStackSynthesizer?: IReusableStackSynthesizer;
+
+  /**
+   * TODO
+   * @default - TODO
+   */
+  readonly validationPlugins?: IValidationPlugin[];
```

Comment:
What about `PolicyValidation`? I really want to avoid using the term `enforcement` anywhere because I don't want anyone to think that this feature is actually _enforcing_ their policies.

Reply:
I'd also suggest `PolicyCompliance` - for your discretion :)

Reply:
@iliapolo I renamed everything to `PolicyValidation`